packages/markdown/package.json

```diff
@@ -8,6 +8,7 @@
     "@readme/syntax-highlighter": "^6.0.3",
     "@readme/variable": "^6.0.3",
     "hast-util-sanitize": "^1.2.0",
+    "isomorphic-style-loader": "^5.1.0",
```

Comment:
Can you move this into being a dev dependency?

Reply:
Yes!